File: cours-03/exercice2.java
void main() {
    Random rng = new Random();
    Scanner scan = new Scanner(System.in);

    int n = rng.nextInt(1, 100);
    int x;
    do {
        System.out.println("Veuillez deviner le nombre :");
        x = scan.nextInt();

        if(x < n) {
            System.out.println("Le nombre est plus grand.");
        }
        if(x > n) {
            System.out.println("Le nombre est plus petit.");
        }
    } while(x != n);

    System.out.println("Vous avez trouvé le nombre!");
}

File: cours-03/exercice2.c
#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main()
{
    srand(time(NULL));
    int n = (rand() % 100) + 1;
    
    int x;
    do {
        printf("Veuillez deviner le nombre : ");
        scanf("%i", &x);
        
        if(x < n) {
            printf("Plus grand.\n");
        }
        if(x > n) {
            printf("Plus petit.\n");
        }
    } while(x != n);
    
    printf("Vous avez deviné le nombre!");

    return 0;
}